src/rosetta-helpers.ts

```diff
@@ -798,25 +800,32 @@ function parseDelegateStxArgs(contract: ContractCallTransaction): RosettaDelegat
     throw new Error(`Could not find field function_args in contract call`);
   }
 
+  // (define-public (delegate-stx
+  //   (amount-ustx uint)
+  //   (delegate-to principal)
+  //   (until-burn-ht (optional uint))
+  //   (pox-addr (optional { version: (buff 1),
+  //                         hashbytes: (buff 20) })))
+
```

Comment:
This seems to have slipped in (also some comments in lines below)

Reply:
It was put in somewhat intentionally to give context to the reader for what the below `function_args` correspond to (since they were changed to get by index rather than unreliable name)